packages/core/src/App/Containers/CitizenshipModal/set-citizenship-form.jsx

```diff
@@ -17,7 +17,6 @@ const SetCitizenshipForm = ({ class_prefix, errors, touched, setFieldTouched, se
                             className={`${class_prefix}__residence-field`}
                             type='text'
                             label={localize('Citizenship')}
-                            error={touched.citizenship && errors.citizenship}
```

Comment:
should we also remove touched error?

Reply:
they mentioned, that there can be no errors at all.
But thanks for your remark. It is way better to have a check for inputed value and make this input's border red, when the value is incorrect or the field was just touched.
will refactor the code.
Thank you so much Amina :+1: